concepts/choosing_a_dataset.md

```diff
@@ -68,6 +68,17 @@ This data is available in the **telemetry_new_profile_parquet** dataset.
 
 {% include "/datasets/batch_view/new_profile/intro.md" %}
 
+# Update Derived Dataset
+
+The [update ping](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/update-ping.html)
+is sent from Firefox Desktop when a browser update is ready to be applied and after it was correctly applied.
+It contains the build information and the update blob information, in addition to some information about the
+user environment.
+
+This data is available in the **telemetry_update_parquet** dataset.
```

Comment:
No need for this line. Accessing the data is discussed in intro.md, which is included below.